app/app.go

```diff
@@ -8,6 +8,21 @@ import (
 	"path/filepath"
 	"strings"
 
+	// "github.com/CosmWasm/wasmd/x/wasm"
+	// wasmkeeper "github.com/CosmWasm/wasmd/x/wasm/keeper"
+
+	"cosmossdk.io/log"
+	// "github.com/CosmWasm/wasmd/x/wasm"
```

Comment:
What's up with the commented out imports?

Reply:
I pushed wasm fixes by uncommenting them in #1039